devmgmtV2/routes/dashboard.routes.js

```diff
@@ -0,0 +1,15 @@
+"use strict";
+
+let {
+    getSystemMemory,
+    getSystemSpace,
+    getSystemCpu,
+    getSystemVersion
+} = require("../controllers/dashboard.controller")
+
+module.exports = app => {
+    app.get("/dashboard/memory", getSystemMemory);
+    app.get("/dashboard/space", getSystemSpace);
+    app.get("/dashboard/cpu", getSystemCpu);
+    app.get("/dashboard/version", getSystemVersion);
```

Comment:
We may need another level of grouping for these metrics.
May change api paths from /dashboard/memory to /dashboard/system/memory etc.